src/UCTSearch.cpp

```diff
@@ -117,7 +117,7 @@ void UCTSearch::dump_stats(KoState & state, UCTNode & parent) {
 
     int movecount = 0;
     for (const auto& node : parent.get_children()) {
-        if (++movecount > 2 && !node->get_visits()) break;
```

Comment:
See comment on why this behaves like it does.

Reply:
Ah yes good idea, in fact I'm tempted to just always display the first move with 0 visits. What do you think? For now I'll just put it back the way it was.